modules/assignment/src/main/java/com/intuit/wasabi/assignment/cache/impl/AssignmentsMetadataCacheRefreshTask.java

```diff
@@ -49,7 +49,8 @@ public AssignmentsMetadataCacheRefreshTask(AssignmentsMetadataCache metadataCach
     @Override
     public void run() {
         try {
-            LOGGER.info("AssignmentsMetadataCache refresh started at = {}", timeService.getCurrentTime());
+            Date startTime = timeService.getCurrentTime();
+            LOGGER.debug("AssignmentsMetadataCache refresh started at = {}", startTime);
```

Comment:
In PROD, App runs at the INFO level. And I wanted to just print a single line highlighting that cache has been refreshed, so kept a single log with INFO level with the time taken to refresh.

This is based on the endurance test experiment where 3 lines were being logged at the cache refresh event, that makes it difficult to understand/track/validate behavior.

I case of any further debugging, we can explicitly enable DEBUG log for this class, which would be rare case scenario mostly in PRE-PROD or LOCAL environment.